tests/GShrinkExample.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE DeriveGeneric, ScopedTypeVariables #-}
+
+module GCoShrinkExample where
```

Comment:
I think this should be `GShrinkExample`, not `GCoShrinkExample`, based on the file name.